ui/router.go

```diff
@@ -65,6 +65,7 @@ func initRouters() {
 	beego.Router("/api/projects/:id([0-9]+)/logs/filter", &api.ProjectAPI{}, "post:FilterAccessLog")
 	beego.Router("/api/users/?:id", &api.UserAPI{})
 	beego.Router("/api/users/:id([0-9]+)/password", &api.UserAPI{}, "put:ChangePassword")
+	beego.Router("/api/registry/sync", &api.RegistryAPI{}, "put:Sync")
```

Comment:
Could you update the url to /api/internal/syncregistry  and let the action be "POST"?

such that we may apply some global policy to /api/internal/***